assets/components/account-header/account-header-example.html

```diff
@@ -11,7 +11,7 @@
     <div class="header-global">
       <div class="header-logo">
         <a href="https://www.gov.uk" title="Go to the GOV.UK homepage" id="logo" class="content">
-          <img src="images/gov.uk_logotype_crown_invert_trans.png?0.19.2" width="36" height="32" alt="">GOV.UK</a>
+          <img src="public/assets/images/gov.uk_logotype_crown_invert_trans.png?0.19.2" width="36" height="32" alt="">GOV.UK</a>
```

Comment:
Does including images like this make them harder to update from GOV.UK?

Reply:
No It doesn't